### ISSUE TITLE
Add datasets counts to publishers page

### ISSUE THREAD

**Issue by kimwdavidson** [GetDKAN/data-catalog-app#68]
## User Story
When I go to the publisher page, I want to see how many datasets are available from each publisher, so that I know how much information I can expect to find from them. 

## Background
This is currently available on DKAN1, and we found in user testing that users accustomed to DKAN1 miss this feature. 

## Acceptance Criteria
- When I go to the publisher page
- Then I see the number of datasets for each publisher on that page

- [x] Working downstream on Marine Scotland /groups
- [x] Working upstream (8.x branch) on /publishers
- [ ] Working upstream to 2.0

## Estimate: 8

**PR**

https://github.com/GetDKAN/data-catalog-components/pull/140

**Comment by cmugisha** [comment]
@alexiscott This ticket has replaced #427. Can you estimate it please?

**Comment by alexiscott** [comment]
@rhabbachi  Not quite there with this for Marine Scotland, but almost. I will send a message one this is ready. Thanks!

**Comment by cmugisha** [comment]
From Alex on March 28th: "Hi @rhabbachi, just a heads up that I have added a PR for Marine Scotland (which should be easier for other sites - I added a description for that to the PR.)  for ticket https://github.com/GetDKAN/data-catalog-app/issues/68 here:
https://github.com/NuCivic/data-marine-scotland/pull/27#pullrequestreview-622774801"

**Comment by rhabbachi** [comment]
Looks goods for marine scotland. @alexiscott do we need to do anything for the 2.x implementation?

**Comment by alexiscott** [comment]
> Looks goods for marine scotland. @alexiscott do we need to do anything for the 2.x implementation?

Yes, we should. I know that @dgading had some thoughts on this as the approach for data in 2.0 is cross app so will work differently.

**Comment by cmugisha** [comment]
@alexiscott @dgading Should we just create a new ticket for the 2.0 implementation or should we keep that in this ticket?

**Comment by cmugisha** [comment]
Moved to the backlog because we need to check with Alex if the 2.0 implementation aspect of this ticket needs to be tracked in the Support or in the Product board.

**Comment by alexiscott** [comment]
@cmugisha , that might be a question for @dgading

**Comment by dgading** [comment]
Closing this ticket for now. We are going through some design revisions and this may not be needed for the new version we are building. If it is needed, we can create a new ticket from the updated designs.
